### PR TITLE
Zachary Prince

### PR DESCRIPTION
**Pull request by z-prince**

### CODE REVIEW

**Review by kaitvan — COMMENTED** [review]
Overall, this is good work. Can you please update your Gear Up link? That's the only thing I'm missing. The rest looks pretty solid.

**Comment by z-prince** [comment]
> Overall, this is good work. Can you please update your Gear Up link? That's the only thing I'm missing. The rest looks pretty solid.

Sorry about that! The link should be updated in the README.md.

File: final_prep/README.md [hunk 1]
```diff
@@ -23,52 +23,54 @@ Some things to consider as you refactor include...
...
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- Add Week 1 Screenshot [Here](https://user-images.githubusercontent.com/65498038/153766905-47e8eb20-f732-419b-a285-02c612077886.png)
+- Add Week 2 Screenshot [Here](https://user-images.githubusercontent.com/65498038/153766926-d892bf4a-bafa-4327-9937-04fba217b35e.png)
+
+- Add Week 3 Screenshot [Here](https://user-images.githubusercontent.com/65498038/153766948-2736839d-2119-4b1a-9098-2aebc87d3591.png)
+
 
 ## Mentorship Prep
```
[the remark this answers is not in the document]
Comment (z-prince) [reply final_prep/README.md:53]: Noted! I don't have my computer at work today so I will have to tackle this when I get home.

File: final_prep/README.md [hunk 2]
```diff
@@ -23,52 +23,54 @@ Some things to consider as you refactor include...
...
 ## Lesson Prep
 
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+- [X] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+  - [X] Add a link to your gist [here](https://gist.github.com/z-prince/12ebfe37a883e65e4e37e4ed7a9fa804)
 
 ## Group Work Prep
```
[the remark this answers is not in the document]
Comment (z-prince) [reply final_prep/README.md:69]: That makes sense. I am completely open to evolving my teamwork ethic and preferences when I get in the groove of things and a better understanding of how the process works.

**Pull request closed without merging**